README.md

```diff
@@ -141,6 +141,33 @@ Output
 
 
 
+## Defining custom steps
+
+I case your flow is different and you would like to define your own wording
+it can be done with:
+
+```ruby
+require "bdd/rspec/example_group_steps"
+
+module ExampleGroupPolish
+  include ExampleGroupSteps
+  define_bdd_step(*%w[Zakładając Jeśli To]) # Polish versions :)
```

Comment:
what can we do to make it like this?

```ruby
require "bdd/rspec"
Bdd::RSpec.define_steps(*%w[Zakładając Jeśli To]) # Polish versions :)
```

don't worry about Minitest for now regarding this.